source/doc-pages/tutorials/todo-app-with-sinatra.md

```diff
@@ -1,232 +0,0 @@
-In this tutorial, we’ll walk through the steps required to build a web API for managing a todo list.
-
-Each example builds on the last, showing how ROM and Sinatra can work together to provide an effective toolkit for rapid prototyping microservices and REST APIs. 
-
-For simplicity and clarity, this tutorial uses ROM’s setup DSL, though in larger apps, we recommend configuring ROM with explicit class definitions. This is analogous to the difference between Sinatra’s routing DSL and its modular application style.
```

Comment:
@nepalez ^ see the above paragraph for at least one take on the differences/use of the ROM setup DSL.